### PR TITLE
Fix: postpone `bodyAfter` display until list items have animated

### LINKED ISSUE #84: The `bodyAfter` element is present whilst items are animating
### Subject of the issue
If the plugin has a `bodyAfter` element and is configured to animate the items in then the `bodyAfter` element visually looks odd as it appears on screen before the items have started / finished animating as the `bodyAfter` element has no animation delay tied to it. 

### Your environment
* Master

### PR DESCRIPTION
**Pull request by danielghost**
Fixes #84.

### Fix
* Display `bodyAfter` once list items have animated. If no animation, it displays immediately as before.

### CODE REVIEW

File: js/ListView.js [hunk 1]
```diff
@@ -45,8 +41,13 @@ class ListView extends ComponentView {
...
    */
   animateListItems() {
-    this.model.getChildren().forEach((listItem, index) => {
+    const items = this.model.getChildren();
+    const itemCount = items.length;
+    items.forEach((listItem, index) => {
       setTimeout(listItem.toggleActive.bind(listItem, true), 200 * index);
+      if (index !== (itemCount - 1)) return;
+      const $item = this.$('.list-item').eq(index);
+      $item.one('transitionend', () => this.$el.removeClass('is-animating'));
```
Comment (oliverfoster) [inline js/ListView.js:50]: This isn't as reliable as it sounds.

As [`transition-property`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property) and [`transition-duration`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-duration) can be a comma separated lists and as [transition events are dispatched for each property](https://drafts.csswg.org/css-transitions/#transition-events), multiple [transitionend](https://developer.mozilla.org/en-US/docs/Web/API/Element/transitionend_event) events are fired if a class applies multiple transitions to different properties of different durations. The can mean that a collective transition animation on a single element is not finished on receipt of the shortest property transition duration, if another has a substantially larger duration.

**Comment by oliverfoster** [comment]
@danielghost please update inline with merged core pr https://github.com/adaptlearning/adapt-contrib-core/pull/537 to require support for fw version 5.39.6 

https://github.com/adaptlearning/adapt_framework/releases/tag/v5.39.6
https://github.com/adaptlearning/adapt-contrib-core/releases/tag/v6.47.0

File: js/ListView.js [hunk 2]
```diff
@@ -45,8 +45,17 @@ class ListView extends ComponentView {
...
+    const itemCount = items.length;
+    items.forEach((listItem, index) => {
+      setTimeout(async () => {
+        listItem.toggleActive(true);
+        if (index !== (itemCount - 1)) return;
+        const $item = this.$('.list-item').eq(index);
+        await transitionsEnded($item);
+        this._hasAnimated = true;
+        this.$el.removeClass('is-animating');
+      }, 200 * index);
```
Comment (oliverfoster) [inline js/ListView.js:58]: Is this for delaying the animation start on each item by a multiple of 200ms?

Comment (danielghost) [reply js/ListView.js:58]: Yes, this is unchanged. The code executed in the timeout has been modified to account for the transition event on the last item, but the actual timings are the same as before (see [line 49](https://github.com/cgkineo/adapt-list/pull/85/files#diff-9e823158aaa858b1b5abd9a2620a11cd6b241ced318439e06743fc13e9523e4cL49))

Comment (oliverfoster) [reply js/ListView.js:58]: Perhaps next issue is moving this into the less. No point changing it in this pr. Oos.

**Pull request merged**